tutorials/CHT/flow-over-plate/buoyantPimpleFoam-laplacianFoam/allrun

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Please rename this to `Allrun` for consistency (and see comment above for moving to another directory).